cli/cmd/generate.go

```diff
@@ -15,6 +17,7 @@ import (
 type AwsGenerateCommandExtraState struct {
 	Output                string
 	UseExistingCloudtrail bool
+	AwsSubAccounts        string
```

Comment:
Any reason not to use []string here?

Reply:
In this context, the value of `AwsSubAccounts` gets sets to the input value from the flag `--aws_subaccounts`; so at this point it's just a string that is used on L236: https://github.com/lacework/go-sdk/blob/86e614b4408858bbc4bb5fc9582b1e4263ffe21b/cli/cmd/generate.go#L236-L240

Reply:
Lets follow the same pattern we have in other CLI commands, instead of having a single flag that requires a CSV of things, we use a flag that can be used multiple times, so instead of:
```
lacework iac --aws_subaccounts "profile:region,profile:region"
```
It would be better to have this instead:
```
lacework iac --aws_subaccount "profile:region" --aws_subaccount "profile:region"
```

Reply:
👍